elspFAQs.html

```diff
@@ -0,0 +1,126 @@
+
+            <div class="letterSampleAccordion">
+                <button class="expandCollapse">
+                    <div>Q1: I am writing a recommendation letter for a student going into a non-physics field, can I still submit my letter for review?</div>
+                </button>
+                <div id="faq1" class="sampleLetter">
+                    <div class="letterContent">
+
+		      <p>Yes, we will review letters for people heading into any field. You will need to be congicent of differing expetations for the institution, which the reviewers may not be aware of. However, these conventions will not impact the reviewers abilities to identify biased language that could be improved.</p>
+                    </div>
+                </div>
+	    </div>
+
+            <div class="letterSampleAccordion">
+                <button class="expandCollapse">
+                    <div>Q2: I am writting a supporting letter for a visa application, can I still submit my letter for review?</div>
+                </button>
+                <div id="faq2" class="sampleLetter">
+                    <div class="letterContent">
+
+		      <p>Yes, this is well within the remitt of recommendation and nomination letters.  Other potential pieces of writing we will review include - but are not limited to - promotion packages, performance reviews, and proposal support letters.</p>
+                    </div>
+                </div>
+	    </div>
+
+            <div class="letterSampleAccordion">
+                <button class="expandCollapse">
+                    <div>Q3: I never have enough advanced notice to submit a letter, how can I get personalized feedback?</div>
+                </button>
+                <div id="faq3" class="sampleLetter">
+                    <div class="letterContent">
+
+		      <p>You can submit a letter you wrote recently for consideration. This can help you identify common words or constructions you use that could be strengthened.</p>
+                    </div>
+                </div>
+	    </div>
+
+            <div class="letterSampleAccordion">
+                <button class="expandCollapse">
+                    <div>Q4: I am not <it>really</it> working in a space or physics field, can I still submit my letter for review?</div>
+                </button>
+                <div id="faq4" class="sampleLetter">
+                    <div class="letterContent">
+
+		      <p>Yes, we will review letters from people working in any discipline. You will need to be congicent of differing expetations for your field, which the reviewers may not be aware of. However, these conventions will not impact the reviewers abilities to identify biased language that could be improved.</p>
+                    </div>
+                </div>
+	    </div>
+
+            <div class="letterSampleAccordion">
+                <button class="expandCollapse">
+                    <div>Q5: I have an idea!</div>
```

Comment:
```suggestion
                    <div>Q4: Do you take suggestions from the community?</div>
```
This is edited to make it a question rather than a comment…